.zuul.d/e2e-prepare.yml

```diff
@@ -0,0 +1,27 @@
+---
+- hosts: primary
+  gather_facts: true
+  tasks:
+    - name: Prepare workspace
+      ansible.builtin.include_role:
+        name: prepare-workspace
```

Comment:
In [base-crc](https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/zuul.d/_jobs-crc.yaml#L12) job ,[ prepare-workspace](https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/playbooks/crc/start.yaml#L11) role is already called here. We can drop this line.

Reply:
oh, nice - lemme drop it.